### ISSUE TITLE
Is there an ETA for AppBar and BoundingBox example?

### ISSUE THREAD

**Issue by jimstack** [Microsoft/MixedRealityToolkit-Unity#1522]
I was under the (quite possibly unwarranted) impression that these capabilities were coming soon after the MRDL merge began, but it seems like the process has stalled. Are there any updates in terms of timeline for these and/or any other features coming from MRDL?

**Comment by Railboy** [comment]
We've been working on multi-pointer support and other input prereqs for bounding box manipulation. Once #1486 is merged into the dev branch we'll create a new feature branch for the bounding box. In the meantime any input or suggestions you have for #1486 are always welcome. (C&V from #1497)

**Comment by NeerajW** [comment]
Duplicate of #1497

**Comment by cre8ivepark** [comment]
@jimstack We opened a pull request for the Bounding Box normal mode with Two hand Gesture.
#1727 
Adjust mode Bounding Box with handles will be added through a different pull request.
